src/components/Webhooks/WebhooksManagement.js

```diff
@@ -140,39 +131,23 @@ class WebhooksManagement extends React.Component{
                 method: 'post',
                 url: 'https://api.badgr.io/o/token',
                 data
-            }).then(res => {
-                console.log(res, res.data.access_token)                
+            }).then(res => {             
                 axios({
                     headers: {
                         'Authorization': `Bearer ${res.data.access_token}`
                     },
                     method: 'get',
                     url: `https://api.badgr.io/v2/issuers`,
         
-                }).then(res1 => {
-                    //console.log(this.state, res1.data.result)                
-                    const len = res1.data.result.filter(r => r.entityId === this.state.id).length;
-                    //console.log("longitud", len)                
-
+                }).then(res1 => {            
```

Comment:
Let's try to avoid using the promise callbacks. Prefer the async and await syntax.